django/bosscore/test/setup_db.py

```diff
@@ -181,9 +181,18 @@ def insert_cloudvolume_test_data(self):
         self.add_collection('col1', 'Description for collection1')
         self.add_coordinate_frame('cf1', 'Description for cf1', 0, 100000, 0, 100000, 0, 100000, 4, 4, 4)
         self.add_experiment('col1', 'exp1', 'cf1', 10, 500, 1)
-        self.add_channel('col1', 'exp1', 'chan1', 0, 0, 'uint8', 'image', storage_type='cloudvol', bucket='bossdb-test-data')
-        self.add_channel('col1', 'exp1', 'chan2', 0, 0, 'uint16', 'image', storage_type='cloudvol', bucket='bossdb-test-data')
-        self.add_channel('col1', 'exp1', 'anno1', 0, 0, 'uint64', 'annotation', ['channel1'], storage_type='cloudvol', bucket='bossdb-test-data')
+        self.add_channel('col1', 'exp1', 'chan1', 0, 0, 'uint8', 'image', 
```

Comment:
Can you add a comment indicating that these are prepopulated channels that were created manually outside the test files?

Reply:
@dxenes1  added the comment.